src/test/java/org/xstefank/check/TestUtilities.java

```diff
@@ -0,0 +1,45 @@
+package org.xstefank.check;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.xstefank.model.yaml.Format;
+import org.xstefank.model.yaml.FormatConfig;
+import org.xstefank.model.yaml.SkipPatterns;
+import java.io.File;
+import java.io.IOException;
+
+public class TestUtilities {
+
+    public static JsonNode readJsonNode(String resource) throws IOException {
+        File testFile = new File(SkipCheck.class.getClassLoader().getResource(resource).getFile());
+        return new ObjectMapper().readTree(testFile);
+    }
+
+    public static FormatConfig setUpFormatConfig(SkipPatterns testSkipPatterns) {
+        Format testFormat = new Format();
+        testFormat.setSkipPatterns(testSkipPatterns);
+        FormatConfig testFormatConfig = new FormatConfig();
+        testFormatConfig.setFormat(testFormat);
+        return testFormatConfig;
+    }
+
+    public static String getStubMethodName() {
```

Comment:
sorry but this is wrong, will StubMethod always be getJsonWithCommits? why to name is as stub method in utils class, when I as a user call `TestUtilities.getStubMethodName` I wouldn't by any chance expect the one exact string returned by this method